kol/onshape/converter.py

```diff
@@ -639,7 +639,7 @@ def get_urdf_part(self, key: Key, joint: Joint | None = None) -> urdf.Link:
         mass = part_dynamic.mass[0]
         if mass <= 0.0:
             logger.error("Part %s has a mass of %f, which is invalid", part_name, mass)
-            mass = 1.0
+            mass = 0.001
```

Comment:
@codekansas - setting a tiny part to actual 1kg is definitely not what we want.
also their inertia is negligible which causes issues in mujoco. I'm in favour of removing such parts completely. Wdyt?